cmd/secret/cmd/root.go

```diff
@@ -223,20 +306,20 @@ func (r *root) serverCmd() *cobra.Command {
 			handler := api.NewMethods(store, logger.Named("handler"))
 			router := chi.NewRouter()
 			srv := &http.Server{Addr: ":" + port, Handler: router}
-
 			router.Use(middleware.Heartbeat("/ping"), middleware.RequestLogger(&middleware.DefaultLogFormatter{
 				Logger: &chiLogger{logger.Named("api")},
 			}))
-			router.Get("/", handler.GetByKey)
 			router.Post("/", handler.SetByKey)
+			router.Get("/", handler.GetByKey)
 
 			done := make(chan os.Signal, 1)
 			shutdownCh := make(chan struct{})
 			signal.Notify(done, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
 
 			go func() {
+				r.logger.Infof("listening ")
```

Comment:
Probably wrong logger is taken
```suggestion
				logger.Infof("listening ")
```